src/main/resources/db/changelog/ddl/0003_create_shedlock_table.xml

```diff
@@ -6,7 +6,7 @@
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
 
   <changeSet id="0002-EIP1-2514-create-shedlock-table" author="fayimora.femi-balogun@valtech.com" context="ddl">
-    <createTable tableName="scheduler-locks">
+    <createTable tableName="shedlock">
```

Comment:
No; you cant do that! It's already been played into at least 1 env (`dev`) - you cant change a liquibase script

Reply:
You'll either need a new liquibase script to rename the table, or change the shedlock config to use the table `scheduler-locks`